config/sizes.json

```diff
@@ -34,5 +34,9 @@
     "handlebars-intl/dist/handlebars-intl.min.js": {
         "bytes": 7267,
         "kbs": 7.1
+    },
+    "ember-intl/dist/ember-intl.min.js": {
+        "bytes": 21362,
+        "kbs": 25
```

Comment:
Oh also, this is done via the Grunt `filesizes` task, so you probably need to add an entry into the `Gruntfile.js` for `ember-intl`'s dist files.